roles/pulp/defaults/main.yml

```diff
@@ -15,7 +15,8 @@ pulp_settings_db_defaults:
       NAME: pulp
       USER: pulp
       PASSWORD: pulp
-pulp_install_dir: '/usr/local/lib/pulp'
+pulp_install_source: pip
+pulp_install_dir: '{{ (pulp_install_source == "packages") | ternary("/usr", "/usr/local/lib/pulp") }}'
```

Comment:
why do we have this difference between sources?

Reply:
Yes, "/usr" if packages (true), "/usr/local/lib/pulp" otherwise (false, "pip").

Reply:
The advantage of doing this in the defaults is that users can still set it.